package/govuk/components/button/_index.scss

```
@@ -166,6 +166,8 @@ $govuk-button-text-colour: govuk-colour("white") !default;
     }
   }
 
+  // @deprecated Disabling buttons using the .govuk-button--disabled class is 
+  // deprecated and will be removed in the next major version.
```

Is it a SassDoc thing where we always have a space below `/// @deprecated` comments?

I've also only seen deprecated mixins or variables so far, how will this show up?

```suggestion
  // deprecated and will be removed in the next major version.

```

---

Do we need to worry about the other selectors being deprecated too?

```css
.govuk-button[disabled="disabled"]
.govuk-button[disabled]
```

Shall we edit ~`package`~ `src` instead and see if the SassDoc output changes?

---

D'oh. Yes, this should be the `src` version of the file rather than `package`. Will resolve.

For deprecating the class, I've followed the format specified in [the managing change documentation](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/managing-change.md#deprecating-a-css-class), which gives the `// @deprecated` format. 

I assume this is intending to copy SassDoc's style without it strictly being SassDoc, so it's easier to find deprecation comments of all types by only searching for '@‍deprecated'.

As this isn't SassDoc in the strict sense, I figured explicitly naming the class in the comment would be enough to indicate that the `[disabled]` variants shouldn't be removed.